### ISSUE TITLE
Flash messages vulnerable to XSS attacks

### ISSUE THREAD

**Issue by Paradoxis** [phalcon/cphalcon#11448]
## Summary 
Adding messages to the `Phalcon\Flash\Session` object will cause the contents of the message to remain unecaped when they are displayed, possibly allowing an attacker to inject HTML into the page (XSS).

* **Class affected:** `\Phalcon\Flash\Session`
* **Method used:** `{{ flashSession.output() }}`

## System details
* **Phalcon version:** 2.0.10
* **OS:** Tested on Ubuntu

## Code examples
```php
// Setting the session in my dependency injection object
$di->set('flashSession, function() {
  return new Session([
        'error'   => 'alert alert-danger',
        'success' => 'alert alert-success',
        'notice'  => 'alert alert-info',
        'warning' => 'alert alert-warning'
    ]);
});
```
```php	
// Adding messages inside of a controller
$this->flashSession->success("<script>alert('This will execute as JavaScript!')</script>");
```
```volt
// Echoing out the messages in a volt template (the message is printed as HTML)
{{ flashSession.output() }} 
```
```volt
// Same result (printed as HTML)
{{ flashSession.output() | escape }}
```
```volt
// Also the same result (printed as HTML)
{% autoescape true %}
  {{ flashSession.output() }}
{% endautoescape %}
```

**Comment by googlle** [comment]
+1

**Comment by sergeyklay** [comment]
Fixed in 2.1.x
